src/std/date.ab

```diff
@@ -50,15 +56,18 @@
 /// %::z   +hh:mm:ss numeric time zone (e.g., -04:00:00)
 /// %:::z  numeric time zone with : to necessary precision (e.g., -04, +05:30)
 /// %Z     alphabetic time zone abbreviation (e.g., EDT)
-///
+/// ```
+/// 
 /// By default, date pads numeric fields with zeroes.  The following optional flags may follow '%':
 ///
+/// ```
 /// -      (hyphen) do not pad the field
 /// _      (underscore) pad with spaces
 /// 0      (zero) pad with zeros
 /// +      pad with zeros, and put '+' before future years with >4 digits
 /// ^      use upper case if possible
 /// #      use opposite case if possible
```

Comment:
Can you add a `\` in front of `#`?
In this way markdown ignore that as header symbol

Reply:
If this is for line 69, it's in a code block, so it should be ignored